gunicorn/app/base.py

```diff
@@ -157,6 +157,9 @@ def load_config(self):
         parser = self.cfg.parser()
         args = parser.parse_args()
 
+        if args.config:
+            self.load_config_from_file(args.config)
+
```

Comment:
This happens later, on line 177. Is there a way we can de-duplicate this?

Reply:
Can we just leave this out, since we already changed `app.wsgiapp.WSGIApplication`.

Reply:
@hramezani I unresolved this thread because I didn't see an answer. Thanks for the continued updates!

Reply:
Sorry, if we remove the second block in line 177, the test ` test_config_file_environment_variable` will fail.
And, if we remove this block one of the newly added tests in this PR(`test_wsgi_app_config[options3-app1:app1]`) will fail.
So, I've decided to keep the first one and change the second one to 
`if env_args.config and not args.config:`